python/foglamp/services/core/api/configuration.py

```diff
@@ -220,77 +220,6 @@
     return web.json_response(result)
 
 
-# async def add_configuration_item(request):
```

Comment:
We document it as something we support, so we should probably reinstate it. Just because the GUI does not use it does not mean other client do not use it. See [JIRA](https://scaledb.atlassian.net/browse/FOGL-1790). Either remove it completely (i.e from routes.py as well) and move code snippet to mentioned JIRA

Reply:
restored and removed comments! per FOGL-1790